### DISCUSSION TITLE
PDF Logo Titlepage

### DISCUSSION THREAD

**Discussion by DavidPara** [quarto-dev/quarto-cli#3841] [Q&A]
I am trying to get a logo on my title page. Here is my YAML:
````

---
title: "First test"
subtitle: "Does this work"
date: "`r format(Sys.time(), '%B %d, %Y')`"
author: "David "
format:
  pdf: 
    include-in-header: 
      text: |
        \usepackage{titling}
        \pretitle{\begin{left}\includegraphics{logo.png}\LARGE\\}
        \posttitle{\end{left}}
        \usepackage{fancyhdr}
        \pagestyle{fancy}
        \fancyhead{}
        \fancyfoot{}
        \fancyfoot[L]{ Confidential - For Internal Use Only}
        \fancyfoot[R]{\thepage}
        \usepackage[T1]{fontenc}
        \usepackage{mathpazo}
        \renewcommand{\familydefault}{ppl}
        \addtokomafont{disposition}{\rmfamily}
        
editor: visual
editor_options: 
  chunk_output_type: console
fontsize: 11pt
geometry: margin=1in
---


````

Here is the rest of the RStudio Code:

````


Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see https://quarto.org/.

Running Code

When you click the Render button a document will be generated that includes both content and the output of embedded code. You can embed code like this:

{r}
#| fig-cap: "My first graph"
#| label: gg
ggplot(mtcars, aes(x=mpg, y=qsec)) + geom_point()

You can add options to executable code like this Figure @gg

{r}
#| echo: false
2 * 2

The echo: false option disables the printing of code (only output is displayed).

\clearpage

more text here


````

Here is the error I am getting:

````
compilation failed- error
Missing $ inserted.
<inserted text> 
                $
l.214 \ifdefined
                \Shaded\renewenvironment{Shaded}{\begin{tcolorbox}[interior ... 

You may need to $ $ around an expression in this file.

````

Any ideas of what is going on?

Also, this takes about 3 minutes to run until this error. Is this a relatively normal amount of time? It makes debugging fairly tedious.

Thanks for your help,

David

**Comment by mcanouil** [comment]
Can you format properly the code?
Use four back ticks to encapsulate code with three back ticks.


`````
````
---
yaml
---

Text

```{r}
code
```

````
`````


Here you are adding some LaTeX code that might conflict with the default Quarto/Pandoc template.
Instead you should consider editing and replacing the ~~title~~ doc-class.tex partial, see <https://quarto.org/docs/journals/templates.html#template-partials>.

Also, you might want to take a look at <https://github.com/nmfs-opensci/quarto_titlepages>.

**Reply by DavidPara** [reply]
Pardon the dumb question, but where are the partial templates stored on a windows system? If I want to replace before-body.tex with something custom, don't I need to put it into the same location? I can't find that file on my PC.

I am using quarto integrated into RStudio.

**Reply by mcanouil** [reply]
Everything is documented in the link I shared about "partials".

**Reply by DavidPara** [reply]
It is clear from that link that I will want to modify title.tex. It also provides links to the source code. It does not say where on the system the original title.tex can be found. For some reason, my search was not working, but i have since found the files in: C:\Program Files\RStudio\resources\app\bin\quarto\share\formats\pdf\pandoc

**Reply by mcanouil** [reply]
Look at https://quarto.org/docs/journals/templates.html#latex-partials especially the part "partials source code here"

**Reply by mcanouil** [reply]
Just in case you are/were thinking to modify the files you found, you should not, you should use a copy where your main Quarto document is.

**Reply by DavidPara** [reply]
Thanks. I might have tried to do  that. I am trying the quarto_titlepages link now.

Interestingly, the error that I posted about is also happening in rmarkdown when I am using the exact code from [https://bookdown.org/yihui/rmarkdown-cookbook/latex-logo.html](url), which is weird.

**Reply by pamelacamejom** [reply]
Did you ever solve this issue?
